spark/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
```

Comment:
It's best practice to pin down the versions of the modules to avoid surprises when building the image sometime in the future.